src/code_80005FD0.c

```diff
@@ -756,92 +764,92 @@ void set_places(void) {
     }
 
     for (playerId = 0; playerId < NUM_PLAYERS; playerId++) {
-        D_801643E0[playerId] = gGPCurrentRaceRankByPlayerId[playerId];
+        gPreviousGPCurrentRaceRankByPlayerId[playerId] = gGPCurrentRaceRankByPlayerId[playerId];
     }
 
-    for (playerId = 0; playerId < var_t4; playerId++) {
-        gGPCurrentRacePlayerIdByRank[playerId] = sp80[playerId];
-        gGPCurrentRaceRankByPlayerId[sp80[playerId]] = playerId;
+    for (playerId = 0; playerId < numPlayer; playerId++) {
+        gGPCurrentRacePlayerIdByRank[playerId] = rankPlayer[playerId];
+        gGPCurrentRaceRankByPlayerId[rankPlayer[playerId]] = playerId;
     }
 
-    for (playerId = 0; playerId < var_t4; playerId++) {
+    for (playerId = 0; playerId < numPlayer; playerId++) {
         a_really_cool_variable_name = D_80164378[playerId];
-        sp80[playerId] = a_really_cool_variable_name;
+        rankPlayer[playerId] = a_really_cool_variable_name;
         gCourseCompletionPercentByRank[playerId] = gCourseCompletionPercentByPlayerId[a_really_cool_variable_name];
     }
 
-    for (playerId = 0; playerId < var_t4 - 1; playerId++) {
-        for (var_t1_3 = playerId + 1; var_t1_3 < var_t4; var_t1_3++) {
+    for (playerId = 0; playerId < numPlayer - 1; playerId++) {
+        for (var_t1_3 = playerId + 1; var_t1_3 < numPlayer; var_t1_3++) {
             if (gCourseCompletionPercentByRank[playerId] < gCourseCompletionPercentByRank[var_t1_3]) {
-                temp_s2 = sp80[playerId];
-                sp80[playerId] = sp80[var_t1_3];
-                sp80[var_t1_3] = temp_s2;
+                temp_s2 = rankPlayer[playerId];
+                rankPlayer[playerId] = rankPlayer[var_t1_3];
+                rankPlayer[var_t1_3] = temp_s2;
                 temp_f0 = gCourseCompletionPercentByRank[playerId];
                 gCourseCompletionPercentByRank[playerId] = gCourseCompletionPercentByRank[var_t1_3];
                 gCourseCompletionPercentByRank[var_t1_3] = temp_f0;
             }
         }
     }
 
-    for (playerId = 0; playerId < var_t4; playerId++) {
-        D_80164408[sp80[playerId]] = playerId;
-        D_80164378[playerId] = sp80[playerId];
+    for (playerId = 0; playerId < numPlayer; playerId++) {
+        gGPCurrentRaceRankByPlayerIdDup[rankPlayer[playerId]] = playerId;
+        D_80164378[playerId] = rankPlayer[playerId];
     }
 }
 
-void func_800070F4(void) {
+void update_places(void) {
```

Comment:
update_ranks ? update_rankings ?
update_player_rankings?

Can we use ranking or rank instead of place? The term place can also mean like "I'm in that place over there" which could be confusing.